src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -1670,9 +1691,10 @@ public async Task<TaxInfo> GetTaxInfoAsync(ISubscriber subscriber)
 
     public async Task SaveTaxInfoAsync(ISubscriber subscriber, TaxInfo taxInfo)
     {
-        if (subscriber != null && !string.IsNullOrWhiteSpace(subscriber.GatewayCustomerId))
-        {
-            var customer = await _stripeAdapter.CustomerUpdateAsync(subscriber.GatewayCustomerId, new CustomerUpdateOptions
+        if (string.IsNullOrWhiteSpace(subscriber?.GatewayCustomerId) || subscriber.IsUser()) return;
```

Comment:
:art: [Control blocks should always use curly braces (even 1-line braces)](https://contributing.bitwarden.com/contributing/code-style/csharp#control-blocks)